server/common/errors.py

```diff
@@ -2,69 +2,60 @@ class FilterError(Exception):
     """
     Raised when filter is malformed
     """
-
-    def __init__(self, message):
-        self.message = message
-
-
-class InteractiveError(Exception):
-    """
-    Raised when computation would exceed interactive time
-    """
-
-    def __init__(self, message):
-        self.message = message
+    pass
 
 
 class JSONEncodingValueError(Exception):
     """
     Raised when file loaded into scanpy is misformatted
     """
-
-    def __init__(self, message):
-        self.message = message
+    pass
 
 
 class MimeTypeError(Exception):
     """
     Raised when incompatible MIME type selected
     """
-
-    def __init__(self, message):
-        self.message = message
+    pass
 
 
 class PrepareError(Exception):
     """
     Raised when data is misprepared
     """
-
-    def __init__(self, message):
-        self.message = message
+    pass
 
 
 class ScanpyFileError(Exception):
```

Comment:
Should adaptor-specific errors be defined in a common flie?

Maybe there should be a more general exception?  For example,  DatasetAccessError or something like that?

Reply:
Added DatasetAccessError, removed ScanpyFileError.